src/vstest.console/Resources/Resources.resx

```diff
@@ -238,6 +238,9 @@
 
       Change the verbosity level for console logger. Allowed values for verbosity: quiet, minimal and normal.
       Example: /logger:console;verbosity=&lt;Defaults to "minimal"&gt;
+	  
```

Comment:
Please provide a link back to the https://github.com/Microsoft/vstest-docs/blob/master/docs/report.md#1-console-logger

Reply:
We should also update syntax and example in https://github.com/Microsoft/vstest-docs/blob/master/docs/report.md#1-console-logger to include prefix.